testing/test-crd-type-a.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -euo pipefail
+
+VALUES_YAML="$( dirname -- ${BASH_SOURCE[0]} )/chart/values.yaml"
+PARENT_PATH=".testTypeA"
+TEST_CASES=$(yq "$PARENT_PATH"' | keys | @tsv' "$VALUES_YAML")
+ERRORS=0
```

Comment:
This seems to be doing a lot of complicated stuff that may break in non obvious ways if those yaml files were to change. Sometimes it's better to just duplicate things (leaving a comment) for the sake of clarity, and stay silly :3